sycl/test-e2e/Matrix/joint_matrix_bf16_fill_k_cache_runtime_dim.cpp

```diff
@@ -1,4 +1,6 @@
 //==--- joint_matrix_bf16_fill_k_cache_runtime_dim.cpp  - DPC++ joint_matrix--------==//
+// XFAIL: arch-intel_gpu_bmg_g21
+// XFAIL-TRACKER: https://github.com/intel/llvm/issues/16922
```

Comment:
xfail placement corrupts license header. Please, move to line 21, to the other xfails. thanks.

Reply:
sorry idk what i was thinking with this one

Reply:
fixed in latest commit